util/s3_repository.go

```diff
@@ -46,11 +47,12 @@ type FilesInfo struct {
 }
 
 func FileInfoHeader() string {
-	return fmt.Sprintf("%-50s %-50s %-25s %-15s", "Name", "Description", "Version", "Created")
+	res := fmt.Sprintf("%-50s %-50s %-15s %-20s %-15s", "Name", "Description", "Version", "Created", "Platform")
+	return strings.TrimSpace(res)
 }
 
 func (f *FileInfo) String() string {
-	return fmt.Sprintf("%-50s %-50s %-25s %-15s", f.Namespace+"/"+f.Name, f.Description, f.Version, f.Created)
+	return fmt.Sprintf("%-50s %-50s %-15s %-20s", f.Namespace+"/"+f.Name, f.Description, f.Version, f.Created)
```

Comment:
You don't have the `f.Platform` here? If not, I think we should also not include it in the header (L51), however it would be great if this could be added here as well as this will allow people to see whether they can expect any issues prior to downloading the package.

Reply:
Let me quickly test this and fix it.

Reply:
Turns out this `String()` is used for **images**, not packages. But the header (L51) is shared among the two. If I correct the header here (see the BUREK):

```go
func FileInfoHeader() string {
  res := fmt.Sprintf("%-50s %-50s %-15s %-20s %-15s", "Name", "Description", "Version", "Created", "PlatformBUREK")
  return strings.TrimSpace(res)
}
```

then I get:

```bash
$ capstan search
Name                                               Description                                        Version         Created              PlatformBUREK
mike/osv-loader                                    OSv Bootloader                                     3e34265         2017-08-17 11:20
```
and also:

```bash
$ capstan package search
Name                                               Description                                        Version         Created              PlatformBUREK
apache.spark-2.1.1                                 Apache Spark                                       0.2             2017-08-17 07:36     Ubuntu-14.04
erlang-7.0                                         Erlang                                             0.1             2017-08-17 08:23     Ubuntu-14.04
eu.mikelangelo-project.app.hadoop-hdfs             Hadoop HDFS                                        2.7.2           N/A
eu.mikelangelo-project.app.hello-node              NodeJS-4.4.5                                       4.4.5           N/A
eu.mikelangelo-project.app.node-4.2.6              NodeJS                                             v0.24-300-g33e3a36 N/A
mysql-5.6.21                                       MySQL 5.6.21.                                      0.1             2017-08-17 08:02     Ubuntu-14.04
node-4.4.5                                         NodeJS 4.4.5                                       0.1             2017-08-17 07:14     Ubuntu-14.04
node-4.8.2                                         NodeJS 4.8.2                                       0.1             2017-08-17 07:43     Ubuntu-14.04
node-6.10.2                                        NodeJS 6.10.2                                      0.1             2017-08-17 07:51     Ubuntu-14.04
ompi-1.10                                          Open MPI                                           0.1             2017-08-17 08:11     Ubuntu-14.04
openfoam.core-2.4.0                                OpenFOAM Core                                      0.2             2017-08-18 12:12     Ubuntu-14.04
openfoam.pimplefoam-2.4.0                          OpenFOAM pimpleFoam                                0.1             2017-08-17 07:16     Ubuntu-14.04
openfoam.pisofoam-2.4.0                            OpenFOAM pisoFoam                                  0.1             2017-08-17 07:38     Ubuntu-14.04
openfoam.poroussimplefoam-2.4.0                    OpenFOAM porousSimpleFoam                          0.1             2017-08-17 08:20     Ubuntu-14.04
openfoam.potentialfoam-2.4.0                       OpenFOAM potentialFoam                             0.1             2017-08-17 07:52     Ubuntu-14.04
openfoam.rhoporoussimplefoam-2.4.0                 OpenFOAM rhoPorousSimpleFoam                       0.1             2017-08-17 07:10     Ubuntu-14.04
openfoam.rhosimplefoam-2.4.0                       OpenFOAM rhoSimpleFoam                             0.1             2017-08-17 08:18     Ubuntu-14.04
openfoam.simplefoam-2.4.0                          OpenFOAM simpleFoam                                0.1             2017-08-17 07:57     Ubuntu-14.04
openjdk7                                           OpenJDK 1.7.0                                      0.1             2017-08-17 07:43     Ubuntu-14.04
openjdk8-zulu-compact1                             OpenJDK 1.8.0_112                                  0.1             2017-08-17 08:16     Ubuntu-14.04
openjdk8-zulu-compact3-with-java-beans             OpenJDK 1.8.0_112 zulu-compact3-with-java-beans    0.1             2017-08-17 07:08     Ubuntu-14.04
osv.bootstrap                                      OSv Bootstrap                                      0.1             2017-08-17 07:56     Ubuntu-14.04
osv.cli                                            OSv Command Line Interface                         0.1             2017-08-17 07:54     Ubuntu-14.04
osv.cloud-init                                     cloud-init                                         0.1             2017-08-17 08:03     Ubuntu-14.04
osv.httpserver                                     OSv HTTP REST Server                               0.1             2017-08-17 07:58     Ubuntu-14.04
osv.nfs                                            OSv NFS Client Tools                               0.1             2017-08-17 07:44     Ubuntu-14.04
python-2.7                                         Python 2.7                                         0.1             2017-08-17 08:15     Ubuntu-14.04
sample-0.1                                         Sample package that does nothing                   0.1             2017-08-17 07:44     Ubuntu-14.04
```

(notice how the header of the last column is `PlatformBUREK` in both cases)

Anyway, the next PR that I already have implemented is "Add 'platform' column to the output of 'capstan images'"  that will also add Platform data to Capstan images, so there will be actual value provided there. May I suggest we leave it as it is?